meilisearch.gemspec

```diff
@@ -16,4 +16,5 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.6.0'
   s.add_dependency 'httparty', '>= 0.17.1', '< 0.21.0'
+  s.add_dependency 'openssl'
```

Comment:
Why did you prefer using `openssl` than the `jwt` bundle?

Reply:
I think it's for `hmac` utility I ask myself the same question

Reply:
Solved in #303 

In the end, I didn't need to require this, it is already "built-in" in the ruby version. :)